mage_ai/frontend/pages/pipelines/index.tsx

```diff
@@ -288,6 +289,40 @@ function PipelineListPage() {
     q,
   ]);
 
+  const [downloadPipeline] = useMutation(
+    ({
+      pipelineUUID,
+      filesOnly = false,
+    }: {
+      pipelineUUID: string;
+      filesOnly?: boolean;
+    }) => 
+      api.downloads.pipelines.useCreate(pipelineUUID)({'download': {'ignore_folder_structure': filesOnly}}),
```

Comment:
linting: spaces between brackets. May also be more readable to format this over multiple lines like this:
```
api.downloads.pipelines.useCreate(pipelineUUID)(
  { 'download': { 'ignore_folder_structure': filesOnly } },
),
```

Reply:
Left the curly brackets next to the parentheses; hope this works.

Reply:
yup, that works too!